internal/controller/storageclaim_controller.go

```diff
@@ -598,6 +678,13 @@ func (r *StorageClaimReconciler) list(obj client.ObjectList, listOptions ...clie
 	return r.Client.List(r.ctx, obj, listOptions...)
 }
 
+func (r *StorageClaimReconciler) create(obj client.Object) error {
+	if err := r.Client.Create(r.ctx, obj); err != nil && !errors.IsNotFound(err) {
```

Comment:
can create return a notFound error?

Reply:
Removed both `create` and `delete` aux funcs post move to k8sutil